src/coreComponents/mainInterface/CMakeLists.txt

```diff
@@ -0,0 +1,53 @@
+#
+# Specify all headers
+#
+set(mainInterface_headers
+    ProblemManager.hpp
+    initialization.hpp
+    GeosxState.hpp
+   )
+
+
+#
+# Specify all sources
+#
+
+set(mainInterface_sources
+    ProblemManager.cpp
+    initialization.cpp
+    GeosxState.cpp
+   )
+
+set( dependencyList events fileIO optionparser RAJA conduit)
```

Comment:
I think the include list in `ProblemManager` is a bit overkill. I know that `fileIO` is dependent on `mesh`, so that dependency should be inherited. And `ProblemManager` should depend on `physicsSolvers`...but it seems to build without the dependency...probably because there is nothing linking with the `mainInterface` that isn't linking with `geosx_core`.